wikipediapreview-wordpress.php

```diff
@@ -15,12 +15,16 @@ function wikipediapreview_enqueue_scripts() {
     $assets_dir = plugin_dir_url( __FILE__ ) . 'assets/';
     
     wp_enqueue_script(
-        'wikipedia-preview', $assets_dir . 'js/wikipedia-preview.production.js', [], false, true 
+        'wikipedia-preview-script', $assets_dir . 'js/wikipedia-preview.production.js', [], false, true 
```

Comment:
Could you document the last 2 params. We need to get them right to make sure there's good caching but the cache is also busted when we release a new version.